### PR TITLE
Change text for button

### PR DESCRIPTION
**Pull request by yosefmare**

### CODE REVIEW

File: src/app/shared/data-table/data-table.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="data-table-wrapper">
   <div class="data-table-header">
-    <button mat-raised-button class="data-table-add-button">Basic</button>
+    <button mat-raised-button class="data-table-add-button"mat-button color="primary" (click)="addButtonClicked()" >Add</button>
```
Comment (EzDor) [inline src/app/shared/data-table/data-table.component.html]: @yosefmare 
Why did you add `mat-button`?

**Review by EzDor — COMMENTED** [review]
@yosefmare 
Did you run the Prettier formatter?

File: src/app/shared/data-table/data-table.component.html (hunk 1 above)
Comment (yosefmare) [reply src/app/shared/data-table/data-table.component.html]: fixed

**Review by EzDor — APPROVED** [review]
